domain/secret/watcher_test.go

```diff
@@ -277,6 +277,111 @@ func (s *watcherSuite) TestWatchObsoleteForUnitsOwned(c *gc.C) {
 	harness.Run(c, []string(nil))
 }
 
+func (s *watcherSuite) TestWatchObsoleteForAppOwnedSecretDeletion(c *gc.C) {
```

Comment:
Do we have coverage for a deleted secret uri being received and removing any existing secret revisions from the change set?